Makefile

```diff
@@ -118,12 +120,15 @@ ami: .artefacts .log vendor/packer vendor/jq $(VENV) ansible/roles/vendor
 		-var 'aws_vpc_id=$(AWS_BUILD_VPC_ID)' \
 		-var 'base_ami_id=$(shell get_base_ami_id $(ROLE))' \
 		-var 'build_name=$(shell get_build_name $(BUILD_NAME))' \
-		-var 'build_version=$(shell get_repo_version)' \
+		-var 'build_version=$(VERSION)-$(VERSION_ROLE)' \
```

Comment:
Do we need to remove the get_repo_version script?

Reply:
yes, yes :) I'm on it...